DRsim/src/DRsimPrimaryGeneratorAction.cc

```diff
@@ -81,13 +81,13 @@ void DRsimPrimaryGeneratorAction::GeneratePrimaries(G4Event* event) {
     G4double yTowerFront = 0;
     G4double zTowerFront = 0;
 
-    if (-fTheta < 0.98) {
+    if (-fTheta < 0.98) { //theta-3.0, if(-fTheta < 1.009){
```

Comment:
What does the comment mean? Is the comment necessary?

Reply:
In Tungsten, We should shot with theta-3.0. when this run, have to change this if loop that divide Barrel and Endcap tower.

Reply:
I see, so the value `0.98` is hard-coded... I think retrieving it from `DRsimDetectorConstruction` (similar to what we did in `DRsimSteppingAction`) would be ideal, but let's leave it for the moment since we don't expect to use this repo for the long term (we'll migrate to [fun4all_eicdetectors](https://github.com/eic/fun4all_eicdetectors/tree/master/simulation/g4simulation/g4drcalo))